### PR TITLE
UIEH-579: Disallow adding multiple date ranges

### PR DESCRIPTION
**Pull request by anastasiaHahanova**
## Purpose
According to [UIEH-579](https://issues.folio.org/browse/UIEH-579) the user won't be able to add multiple date range in package-edit and package-create.

## Additional
This PR resolves [UIEH-580](https://issues.folio.org/browse/UIEH-580).

## Screenshots
screen shot 2018-12-06 at 18 42 42
screen shot 2018-12-06 at 18 43 51

### CODE REVIEW

File: src/components/package/_fields/custom-coverage/package-coverage-fields.js [hunk 1]
```diff
@@ -2,39 +2,55 @@ import React, { Component, Fragment } from 'react';
...
     let dateFormat = moment.localeData()._longDateFormat.L;
     let errors;
 
     if (value && !moment.utc(value).isValid()) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.dateRange.format" values={{ dateFormat }} />;
+      errors = (
+        <FormattedMessage
+          id="ui-eholdings.validate.errors.dateRange.format"
+          values={{ dateFormat }}
+        />);
```
Comment (VictorSoroka1) [inline src/components/package/_fields/custom-coverage/package-coverage-fields.js]: Please move `);` on the new line.

File: src/components/package/_fields/custom-coverage/package-coverage-fields.js [hunk 2]
```diff
@@ -59,28 +75,65 @@ class PackageCoverageFields extends Component {
...
+
+    const hasAddButton = coverageDateAmount === 0 || (coverageDateAmount === 1 && !initialValue[0]);
 
+    return (
+      <RepeatableField
+        addLabel={hasAddButton
+          ? <FormattedMessage id="ui-eholdings.package.coverage.addDateRange" />
+          : null}
+        emptyMessage={
+          initialValue.length > 0 && initialValue[0].beginCoverage ?
```
Comment (VictorSoroka1) [inline src/components/package/_fields/custom-coverage/package-coverage-fields.js]: Please move `initialValue.length > 0 && initialValue[0].beginCoverage` to a variable.

File: src/components/package/_fields/custom-coverage/package-coverage-fields.js [hunk 3]
```diff
@@ -59,28 +75,65 @@ class PackageCoverageFields extends Component {
...
+      this.setState((prevState) => ({
+        coverageDateAmount: prevState.coverageDateAmount - 1,
+      }));
+    };
+
+    const hasAddButton = coverageDateAmount === 0 || (coverageDateAmount === 1 && !initialValue[0]);
 
+    return (
+      <RepeatableField
+        addLabel={hasAddButton
```
Comment (VictorSoroka1) [inline src/components/package/_fields/custom-coverage/package-coverage-fields.js]: Please format this one as you did for empty message

```suggestion
        addLabel={
          hasAddButton
            ? <FormattedMessage id="ui-eholdings.package.coverage.addDateRange" />
            : null
        }
```

File: src/components/package/_fields/custom-coverage/package-coverage-fields.js [hunk 4]
```diff
@@ -2,39 +2,55 @@ import React, { Component, Fragment } from 'react';
...
+          values={{ dateFormat }}
+        />);
     }
 
     return errors;
   }
 
   renderField = (dateRange) => {
+    const formatField = (value) => (value ? moment.utc(value) : '');
+
```
Comment (MaksymDryha) [inline src/components/package/_fields/custom-coverage/package-coverage-fields.js]: There is no need to wrap a single parameter with parentheses. The same about the function's body here

Comment (anastasiaHahanova) [reply src/components/package/_fields/custom-coverage/package-coverage-fields.js]: @MaksymDryha, Unfortunately ESLint does not allow me to remove parentheses from the body.

File: src/components/package/_fields/custom-coverage/package-coverage-fields.js [hunk 5]
```diff
@@ -2,39 +2,56 @@ import React, { Component, Fragment } from 'react';
...
 
   static defaultProps = {
-    initialValue: []
+    initialValue: [],
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = { coverageDateAmount: 1 };
```
Comment (Godlevskyi) [inline src/components/package/_fields/custom-coverage/package-coverage-fields.js]: Please use constant here

**Pull request merged**